NitroxClient/Communication/Packets/Processors/ServerStoppedProcessor.cs

```diff
@@ -0,0 +1,17 @@
+﻿using NitroxClient.Communication.Packets.Processors.Abstract;
+using NitroxClient.MonoBehaviours.Gui.InGame;
+using NitroxModel.Packets;
+
+namespace NitroxClient.Communication.Packets.Processors;
+
+public class ServerStoppedProcessor : ClientPacketProcessor<ServerStopped>
+{
+    public override void Process(ServerStopped packet)
+    {
+        // This packet is just meant to tell the modal to modify its message
+        if (LostConnectionModal.Instance)
+        {
+            LostConnectionModal.ServerStopped = true;
+        }
```

Comment:
Instead of botching into the LostConnectionModal we should create a ServerStoppedModal. To prevent the LostConnectionModa from firing we can disconnect the client properly beforehand (but that needs investigation if something will break).